peoples_budget/forms.py

```diff
@@ -5,10 +5,8 @@ class ChangeBudgetForm(forms.Form):
     """Hidden form to store user budget data"""
     json_data = forms.CharField(widget=forms.HiddenInput(), required=True)
 
-
 class SubmitBudgetForm(forms.Form):
     """Combine user budget data and PII"""
     email = forms.EmailField(required=True)
-    address = forms.CharField(required=True)
+    ward = forms.CharField(required=True)
```

Comment:
I noticed that if a user puts a non-integer value into the ward field at /submit ; they'll receive an ValueError at /store-data/ ; 

 we could enforce that the field to only accept integers more elegantly to the user by changing the 
the input type for that particular element from 'text' to integer' (ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number)

I think that can be changed by modifying this field type from CharField to IntegerField  (ref: https://docs.djangoproject.com/en/3.1/ref/forms/fields/#integerfield) ?

Reply:
(not sure if I did this review right here on github?!)

Reply:
Makes sense, I had initially tried using IntegerField and wasn't sure how it looked since it was a dropdown box, but it looks like its only a dropdown on hover